python/ray/_private/runtime_env/conda_utils.py

```diff
@@ -186,7 +186,7 @@ def exec_cmd(
 
 
 def exec_cmd_stream_to_logger(
-    cmd: List[str], logger: logging.Logger, n_lines: int = 10
+    cmd: List[str], logger: logging.Logger, n_lines: int = 50
```

Comment:
@architkulkarni I increased this because last 10 lines are not enough to display the whole error logs. 

I also wonder if we should not prune this at all. Have you seen really long lines from here before?

Reply:
I think it's fine not to prune it.  It's just the error log, not the entire conda install log, right?  Then I think it's okay.  cc @edoakes do you remember if there was some special reason we added the pruning?

Reply:
IIRC it returns the whole conda install log, which can be super long. Pruning it less sounds ok to me